doc/lexer-rules.md

```diff
@@ -308,3 +308,16 @@ As of 4.5, you can also define channel names like enumerations with the followin
 ```
 channels { WSCHANNEL, MYHIDDEN }
 ```
+
+## Lexer Rule Options
+
+### caseInsensitive
```

Comment:
I'm not sure about the best location for `caseInsensitive` lexer rule. Maybe it makes sense to move to https://github.com/antlr/antlr4/blob/master/doc/options.md#rule-options

Reply:
maybe a reference in one spot to the other?

Reply:
Done, I made a reference in options.md: https://github.com/antlr/antlr4/pull/3437/files#diff-6420f854d462e04fe86b98f8bd495e3ec6f42f94121c39a37d228ba74ea91052R114